create-interactive-prototype/manifest.json

```diff
@@ -4,7 +4,6 @@
     "code": "plugin.js",
     "description": "This is a create interactive prototype plugin",
     "permissions": [
-      "content:read",
```

Comment:
penpot.getPage(), needs content:read

Reply:
getPage is outdated, i'll replace it with currentPage that doesn't need content:read